src/main/kotlin/lotto/domin/Lotto.kt

```diff
@@ -0,0 +1,21 @@
+package lotto.domin
+
+import lotto.util.LottoNumberGenerator
+
+class Lotto(
+    private val payment: Int,
+    private val lottoNumberGenerator: LottoNumberGenerator
+) {
+
+    fun issuanceCount(): Int {
+        return payment.div(LOTTO_PRICE)
+    }
+
+    fun issuance(): List<Int> {
+        return this.lottoNumberGenerator.numberSet()
```

Comment:
```suggestion
        return lottoNumberGenerator.numberSet()
```
this는 생략이 가능해요. (이하 유사 관련 내용 동일)

Reply:
개인적으로 this를 사용해서 해당 인스터스를 가리키는 것을 명시해주는 것이 좋다고 보는데 어떻게 생각하시나요?

Reply:
저도 현석님이 해주신 말씀에 동의합니다,,! 답변 감사합니다 🙏